packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -228,14 +232,14 @@ $Calendar-dayPicker-margin: 10px;
 
 .dayIsFocused:active,
 .dayIsHighlighted {
-  background: $ms-color-themeLight;
+  background-color: $ms-color-themeLight;
 }
 
 .dayIsFocused:active,
 .dayIsHighlighted {
   &.day {
-    color: $ms-color-neutralPrimary;
-    font-weight: $ms-font-weight-semibold;
+    color: $ms-color-neutralDark;
+    background-color: $ms-color-neutralLight;
```

Comment:
I think you need to add this to .dayPress as well to fix the month selection issue